DataParser/dataParser.py

```diff
@@ -0,0 +1,138 @@
+import os
+import json
+import xmltodict
+import zipfile
+import pandas as pd
+from pathlib import Path
+import pytest
+import sys
+import filecmp
+
+
+class ooxml_file:
+	"""
+	OOXML file class object to load various XMLs from the zip of the OOXML file and 
+	saving them in json format in the coresponding path in the generate folder.
+	"""
+	def __init__(self,filename):
+		self.filename = filename
+		
+	def load_data(self):
+		"""
+		This function loads the OOXML files, Unzips then and saves their data 
+		in a dictionary format for all the XMLs present in the code.
+		"""
+		archive = zipfile.ZipFile(self.filename, 'r')
+
+		self.xml_files = {}
+		self.rel_files = {}
+		self.other_data_files = {}
+		
+		for zip_obj in archive.infolist():
+			
+			extension = str(zip_obj.filename).strip().split('.')[-1]
+			
+			if extension=='xml':
+				with archive.open(zip_obj.filename) as myfile:
+					data = myfile.read()
+					self.xml_files[zip_obj.filename]= xmltodict.parse(data)
+			
+			elif extension=='rels':
+				with archive.open(zip_obj.filename) as myfile:
+					data = myfile.read()
+					self.rel_files[zip_obj.filename]=xmltodict.parse(data)
+			
+			else:
+				self.other_data_files[zip_obj.filename]=zip_obj.filename
+	
+	def save_json_data(self,out_path):
+		"""
+		This saves the loaded XML file's dictionaries into the path provides 
+		in JSON format.
+
+		@Param out_path: path to salve all the XML files by creating a folder 
+						for the XML file.
+
+		"""
+		path_name_decomp = self.filename.split('/')
+		dir_name = self.filename[:-len(path_name_decomp[-1])]
+		
+		base_name = path_name_decomp[-1].replace('.','_')
+		dir_name = out_path
+
+		print("Processing : ",dir_name,base_name)
```

Comment:
Try to add the processing info in a file - example: DEBUG.txt - instead of relying only on print statements. (You could have both if needed as well)